topos/features/v1/features.proto

```diff
@@ -0,0 +1,283 @@
+    option (google.api.http) = {
+      post : "/v1/{feature_set.name=feature_sets/*}:set"
+      body : "feature_set"
+    };
+  }
+
+  // Deletes a feature set
+  rpc DeleteFeatureSet(DeleteFeatureSetRequest)
+      returns (google.protobuf.Empty) {
+    option (google.api.http) = {
+      delete : "/v1/{name=feature_sets/*}"
+    };
+  }
+
+  // Sets a feature set and a feature relation.
+  rpc SetFeatureSetFeature(FeatureSetFeature) returns (FeatureSetFeature) {
+    option (google.api.http) = {
+      post : "/v1/{feature_set_feature.feature_set_name=feature_sets/*}/"
+             "{feature_set_feature.feature_name=features/*}:set"
+      body : "feature_set_feature"
+    };
+  }
+
+  // Deletes a feature set and a feature relation.
+  rpc DeleteFeatureSetFeature(FeatureSetFeature)
+      returns (google.protobuf.Empty) {
+    option (google.api.http) = {
+      delete : "/v1/{feature_set_feature.feature_set_name=feature_sets/*}/"
+               "{feature_set_feature.feature_name=features/*}"
+    };
+  }
+
+  // Lists feature sets.
+  rpc ListFeatureSets(ListFeatureSetsRequest)
+      returns (ListFeatureSetsResponse) {
+    option (google.api.http) = {
+      get : "/v1/feature_sets"
+    };
+  }
+
+  // Lists feature meta.
+  rpc ListFeatures(ListFeaturesRequest) returns (ListFeaturesResponse) {
+    option (google.api.http) = {
+      get : "/v1/features"
+    };
+  }
+
+  // Sets a feature meta.
+  rpc SetFeature(SetFeatureRequest) returns (Feature) {
+    option (google.api.http) = {
+      post : "/v1/{feature.name=features/*}:set"
+      body : "feature"
+    };
+  }
+
+  // Deletes a feature meta
+  rpc DeleteFeature(DeleteFeatureRequest) returns (google.protobuf.Empty) {
+    option (google.api.http) = {
+      delete : "/v1/{name=features/*}"
+    };
+  }
+
+  // Sets a feature value.
+  rpc SetFeatureValue(SetFeatureValueRequest) returns (FeatureValue) {
+    option (google.api.http) = {
+      post : "/v1/{feature_value.name=feature_values/*}:set"
+      body : "feature_value"
+    };
+  }
+
+  // Sets a batch of feature values.
+  rpc BatchSetFeatureValue(BatchSetFeatureValueRequest)
+      returns (BatchSetFeatureValueResponse) {
+    option (google.api.http) = {
+      post : "/v1/feature_values:batch_set"
+      body : "*"
+    };
+  }
+
+  // Deletes a feature value.
+  rpc DeleteFeatureValue(DeleteFeatureValueRequest)
+      returns (google.protobuf.Empty) {
+    option (google.api.http) = {
+      delete : "/v1/{name=feature_values/*}"
+    };
+  }
+
+  // Search feature values.
+  rpc SearchFeatureValues(SearchFeatureValuesRequest)
+      returns (SearchFeatureValuesResponse) {
+    option (google.api.http) = {
+      post : "/v1/feature_values:search"
+      body : "*"
+    };
+  }
+
+  // Gets Time series of feature values
+  rpc TimeSeriesFeatureValues(TimeSeriesFeatureValuesRequest)
+      returns (TimeSeriesFeatureValuesResponse) {
+    option (google.api.http) = {
+      post : "/v1/feature_values:time_series"
```

Comment:
The [custom HTTP verb](https://cloud.google.com/apis/design/custom_methods#http_mapping) (`time_series` here) should also use lowerCamel case.